openquake/server/tests/tests.py

```diff
@@ -282,3 +286,35 @@ def test_validate_nrml_missing_parameter(self):
         self.assertEqual(resp.status_code, 400)
         self.assertEqual(resp.content,
                          b'Please provide the "xml_text" parameter')
+
+    def test_check_fs_access(self):
+        with tempfile.NamedTemporaryFile(buffering=0, prefix='oq-test_') as f:
+            filename = f.name
```

Comment:
Using buffering works only in Python 3. Should we block this until we have Python 3 packages?